spinner/build.sh

```diff
@@ -52,9 +52,9 @@ function build_service_liferay {
 
 	if [ "$(git -C "${SPINNER_LIFERAY_LXC_REPOSITORY_DIR}" rev-parse --abbrev-ref HEAD)" == "master" ]
 	then
-		liferay_image=$(retrieve_descriptor_data "liferay-image")
+		liferay_image=$(get_environment_descriptor "liferay-image")
```

Comment:
@pedropaccola, there is no ticket prefix (SRE-6066) in the commit message as the previous commit messages, can you please add it?